fe/fe-core/src/main/java/com/starrocks/connector/delta/DeltaLakeFileStats.java

```diff
@@ -74,26 +81,28 @@ public DeltaLakeFileStats(StructType schema, Set<String> nonPartitionPrimitiveCo
             this.corruptedStats = null;
             this.hasValidColumnMetrics = false;
         } else {
-            this.minValues = fileStat.minValues.entrySet().stream()
-                    .filter(e -> nonPartitionPrimitiveColumns.contains(e.getKey()))
-                    .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
-            this.maxValues = fileStat.maxValues.entrySet().stream()
-                    .filter(e -> nonPartitionPrimitiveColumns.contains(e.getKey()))
-                    .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
-            this.nullCounts = fileStat.nullCount.entrySet().stream()
-                    .filter(e -> nonPartitionPrimitiveColumns.contains(e.getKey()))
-                    .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
-            this.corruptedStats = nonPartitionPrimitiveColumns.stream()
-                    .filter(col -> !minValues.containsKey(col) &&
-                            (!nullCounts.containsKey(col) || ((Double) nullCounts.get(col)).longValue() != recordCount))
-                    .collect(Collectors.toSet());
+            this.minValues = new HashMap<>();
```

Comment:
Partition column ndv can be accurately calculated.

Reply:
Good idea, i will optimizer later, but not in the pr.